tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -222,7 +269,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 options: {
                     moduleSource: {
                         func: "fluid.tests.tooltipModuleSource",
-                        args: "{pager}"
+                        args: ["{pager}", [2, 3, 4]]
```

Comment:
This discussion was picked up in the [IRC channel](https://botbot.me/freenode/fluid-work/2016-05-31/?msg=67025052&page=1).